cmd/gaia/cli_test/cli_test.go

```diff
@@ -515,6 +516,48 @@ func TestGaiaCLISendGenerateSignAndBroadcast(t *testing.T) {
 	require.Equal(t, int64(40), fooAcc.GetCoins().AmountOf("steak").Int64())
 }
 
+func TestGaiaCLIConfig(t *testing.T) {
```

Comment:
have you run `make test_cli` with it passing this doesn't run on the CI as far as I can tell

Reply:
dope - just ran it locally and it passed